docs/cloud/faq/sla.md

```diff
@@ -0,0 +1,23 @@
+# Service Level Agreement & Security
+
+## What are the SLA ?
+
+Read more in our [Service Level Agreement](https://www.eventstore.com/cloud-services-service-level-agreement) document
+
+## Does EventStore Ltd have security policy
+
+EventStore Ltd.  has a security policy.  
+The security hasn't been independently audited yet.
+We plan to achieve SOC 2 and ISO 27001 by second half 2021.
```

Comment:
```suggestion
We plan to achieve SOC 2 and ISO 27001 compliance by the second half of 2021.
```

Reply:
Might be good add links to `SOC 2` and `ISO 27001` for people that don't know what it is.

Reply:
dunno, if someone is looking for that specifically, they'll know, otherwise it's some checklist.

Reply:
once we have them, we surely must link to the proof of it

Reply:
I think that we should at least link some reference sites with an explanation.

Reply:
I really don't think so, check in the Azure doc: they link to their certification, not the details of what it is , 
if someone looks for this, they already know what it is about & they need to ✔some checklist.
Will add that they need to reach out to us if they want more details .